pkg/nfd-master/nfd-master.go

```diff
@@ -73,6 +72,7 @@ type Annotations map[string]string
 
 // Command line arguments
 type Args struct {
+	Instance       string
```

Comment:
nit: do we want to keep all items in an alphabetical order?

Reply:
Yes, certainly, thanks for noting this. Fixed